examples/i2c-scanner/laze.yml

```
@@ -5,3 +5,4 @@ apps:
       - bbc-microbit-v2
       - st-nucleo-f042k6
       - st-steval-mkboxpro
+      - heltec-v3
```

Should preserve lexicographical order, as you did in other cases:

```suggestion
      - bbc-microbit-v2
      - heltec-wifi-lora-32-v3
      - st-nucleo-f042k6
      - st-steval-mkboxpro
```

---

The board is known as "Heltec v3" by both Meshtastic https://flasher.meshtastic.org/ and MeshCore https://flasher.meshcore.co.uk/ - should this be preserved?